Source/help.cpp

```diff
@@ -101,7 +101,7 @@ std::vector<std::string> HelpTextLines;
 void InitHelp()
 {
 	HelpFlag = false;
-	char tempString[512];

```

Comment:
There might be other character arrays that need to be doubled in size to accommodate the larger characters. I'd search for `strcpy` and `strcat`. Some of them, like `tempstr` seem large enough as is; others I'm not sure about.

Reply:
I would prefer to do that in a later update, and maybe search them out by testing (ASAN reports them at runtime)